lib/std/mem.zig

```diff
@@ -10,9 +10,19 @@ const testing = std.testing;
 const Endian = std.builtin.Endian;
 const native_endian = builtin.cpu.arch.endian();
 
+/// Runtime known minimum page size.
+/// If the page size cannot be determined at runtime,
```

Comment:
> If the page size cannot be determined at runtime,

Why not `@compileError`? This forces the user to handle extra complexity to rule out, ie comptime calling context.

Reply:
Because the comptime version is a sufficient fallback. Imo, it makes more sense than to throw an error. Always could break it into 2 functions, one which throws, the other which falls back.

Reply:
I added options to allow for different sanity checks.